src/StudentsController.ts

```diff
@@ -98,6 +98,10 @@ export class StudentsController {
     }
 
     public async getOnboardingDataForStudent(student: Student): Promise<Result<{ pdf: Buffer; png: Buffer; link: string }>> {
+        if (!student.correspondingRelationshipTemplateId || !student.givenname || !student.surname) {
+            throw new ApplicationError("error.schoolModule.deletedStudent", "The student seems to be deleted.");
```

Comment:
"deletedStudent" sounds as if the student was deleted with this request.

```suggestion
            throw new ApplicationError("error.schoolModule.studentAlreadyDeleted", "The student seems to be deleted.");
```